concordia/templates/transcriptions/asset_detail.html

```diff
@@ -517,6 +518,16 @@ <h5 id="captcha-modal-title" class="modal-title">Please confirm you are not a ro
 <script src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/2.4.0/openseadragon.min.js" integrity="sha256-grVJpqgDSGBx1q2llmvY+J5zU9hEHbO7UvftFY2bK1w=" crossorigin="anonymous"></script>
 <script src="{% static 'js/contribute.js' %}"></script>
 <script src="{% static 'js/asset-reservation.js' %}"></script>
+<script>
+    if(localStorage.getItem("no-interface-banner")==null) {
```

Comment:
I'm wondering whether this should be less strictly a null check:

```suggestion
    if(!localStorage.getItem("no-interface-banner")) {
```

Reply:
I haven't tested in awhile but wanted to confirm that this doesn't trigger a prompt by default in any of our supported browsers. It's possible that it'll fail for other reasons but [the code to detect that is a bit complex](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Testing_for_availability) and I think the failure mode in this case is probably better than trying something using cookies.

Reply:
I confirmed that this doesn't affect any of our supported browsers but confirming that on mobile reminded me that we might want to leave it hidden when “mobile” is in the user-agent and/or the screen size is below the 1024x768 threshold we're currently focused on:

https://github.com/LibraryOfCongress/concordia/blob/fc5cc5da4787280431592d5f46b7e81b5495baac/concordia/templates/action-app.html#L27

Reply:
Done